### PR TITLE
Latest API Reference

### PR DESCRIPTION
**Pull request by dustinlarimer**
## What does this PR do?

* Adds the latest API Reference swagger file
* Adds previous versions (for continuity)
* Updates the instructions for running the API Ref generator script

### CODE REVIEW

File: lib/swagger/README.md [hunk 1]
```diff
@@ -28,6 +30,14 @@ Run the following script to produce a target swagger file that contains the addi
 ./lib/swagger/build-swagger.sh ./lib/swagger/revisions/spec-20221201.json ./static/swagger/spec.json
 ```
 
-## Validate target file
+## Validate Target File
 
-Once your final swagger file has been created, validate it at [editor.swagger.io](https://editor.swagger.io/). Verify that all the expected resources are presented correctly.
\ No newline at end of file
+Once the target swagger file has been created, validate it at [editor.swagger.io](https://editor.swagger.io/). Verify that all the expected resources are presented correctly.
```
Comment (diana-macrometa) [inline lib/swagger/README.md]:
```suggestion
Once the target Swagger file has been created, validate it at [editor.swagger.io](https://editor.swagger.io/). Verify that all the expected resources are presented correctly.
```

File: lib/swagger/README.md [hunk 2]
```diff
@@ -10,16 +10,18 @@ This script requires the [jq](https://stedolan.github.io/jq/) command line JSON
...
-└── spec-20221201.json
+|── spec-0.17.10-20220823.json
+|── spec-0.17.11-20221019.json
+└── spec-0.17.12-20221214.json
 ```
 
-## Produce target file
+## Produce Target File
 
 Run the following script to produce a target swagger file that contains the additional fields required for our API Reference.
```
Comment (diana-macrometa) [inline lib/swagger/README.md]:
```suggestion
Run the following script to produce a target Swagger file that contains the additional fields required for our API Reference.
```

**Review by diana-macrometa — APPROVED** [review]

**Comment by diana-macrometa** [comment]
Made a few minor suggestions. If you could remove the Transactions section and the internal-only schemas as well, that would be amazing.

**Review by naimishvora-macrometa — APPROVED** [review]
Looks good, I compared swagger with https://github.com/Macrometacorp/c8apidocs/blob/0.17.12/c8_api/swagger.json using a diff checker both look similar.

**Pull request merged**